src/cc1101.h

```diff
@@ -15,58 +15,67 @@
 
 extern char IB_1[14];
 
+// only MAPLE_Mini
+#ifdef MAPLE_Mini
+	extern uint8_t radionr;
+#endif
 
 
-#if defined(ESP8266) || defined(ESP32)
+#if defined(ESP8266) || defined(ESP32) || defined(MAPLE_Mini)
 #include <SPI.h>
 #endif
 namespace cc1101 {
 	/*
 	#ifdef ARDUINO_AVR_ICT_BOARDS_ICT_BOARDS_AVR_RADINOCC1101
-	#define SS					  8  
-	#define PIN_MARK433			  4  // LOW -> 433Mhz | HIGH -> 868Mhz
-	
+	#define SS                    8
+	#define PIN_MARK433           4  // LOW -> 433Mhz | HIGH -> 868Mhz
+
 	#elif ARDUINO_ATMEGA328P_MINICUL  // 8Mhz 
-	#define PIN_MARK433			  0
+	#define PIN_MARK433           0
 	#endif
-				
 	*/
-						
 
-	#define csPin	SS	   // CSN  out
+#ifdef MAPLE_Mini
+	const uint8_t radioCsPin[] = {31, 12, 15, 3};  // PINs from Circuit board for 4 cc110x
```

Comment:
Was genau soll das Bedeuten? PINs from Circuit board 4 cc110x?

Reply:
Das bedeutet, wenn die Platine mit den 4 Empfängern zum Einsatz kommt, das dort jeweils ein anderer Pin genutzt wird. Genau genommen, handelt es sich bei Circuit Board 4 cc110x um die Platine welche im Forum verteilt / erworben werden konnte.

Reply:
Ah, kenne ich nicht.
Vier Empfänger an einer MCU ? Hmm, das wird vom Code ja bei weitem nicht unterstützt.

Reply:
Ja von einer MCU. Code... derzeit wird es nicht unterstützt weil ich erstmal die Kompatibilität erreichen wollte mit einem Empfänger. Die Erweiterung auf mehr Empfänger geht und steht auf einem Blatt in ferner Zukunft... Daher würde ich die Option eben offen lassen.

Reply:
OK habe ich verstanden.

Reply:
Ich habe diesbezüglich in der kommenden Änderung eine Quelle verankert.
https://github.com/HomeAutoUser/SIGNALDuino/commit/02b94bce3a233d27671ee6d75639358fcef02ba3

Reply:
ok, sollten wir dann aber noch mal genauer überlegen, wo wir den Kram hinpacken?
Also ich meine, an welcher Stelle wir die PIns definieren die wir auch benutzen.

Reply:
Eine Option wäre, diese CS Pin Def. dort zu belassen weil für den cc1101 diese PINS ja auch alle hier defieniert werden.